lib/Unicode/Char/General.hs

```diff
@@ -428,6 +431,29 @@ isSymbol c = case generalCategory c of
     OtherSymbol    -> True
     _              -> False
 
+-- | Returns 'True' for alphabetic Unicode characters (lower-case, upper-case
+-- and title-case letters, plus letters of caseless scripts and modifiers
+-- letters).
+--
+-- prop> isLetter c == Data.Char.isLetter c
```

Comment:
This does not hold.

Reply:
This is a deprecated function. I just readded it unchanged and deprecated it instead of removing it.